src/applications/vre/28-8832/wizard/pages/VRECounselorNotification.jsx

```diff
@@ -5,17 +5,23 @@ import { VRE_COUNSELOR_ROOT_URL } from '../../constants';
 const VRECounselorNotification = () => {
   useEffect(() => {
     recordEvent({
-      event: `howToWizard-notice-displayed`,
+      event: `howToWizard-alert-displayed`,
       'reason-for-notice': 'ineligibility - eligible for chapter 31',
     });
   });
+
+  const handleClick = () => {
+    recordEvent({
```

Comment:
Does page navigation need to wait until the recordEvent method completes?

Reply:
@Clark-W-Griswold no, the analytics team has tested this and we do not need to pause execution